runtime/compiler/runtime/J9Profiler.cpp

```diff
@@ -3107,7 +3107,12 @@ TR_JProfilerThread::processWorkingQueue()
 
       _jProfilerMonitor->exit();
 
-      TR_PersistentProfileInfo **prevPtr = &_listHead;
+      /*
```

Comment:
I think this comment  needs little bit more clarification.

Reply:
I added that comment because I felt that `TR_PersistentProfileInfo * volatile *` is hard to read and isn't super clear on what exactly the `volatile` keyword is doing. The comment is there to explicitly state which part is volatile. I'll see if I can make it a bit more clear.

Reply:
I edited the comment to make it a bit more detailed.